osl-operator.yaml

```diff
@@ -0,0 +1,3205 @@
+apiVersion: v1
```

Comment:
You can remove this file, the prod one should only be installed via OLM.

Reply:
I think they should install a subscription with the custom image or run the operator locally interacting with the cluster directly.

Reply:
I removed that file